ee/clickhouse/sql/person.py

```diff
@@ -65,6 +65,13 @@
 {query}
 """
 
+GET_LATEST_PERSON_DISTINCT_ID_SQL = """
+SELECT * FROM person_distinct_id JOIN (
+    SELECT distinct_id, max(_offset) as _offset FROM person_distinct_id WHERE team_id = %(team_id)s GROUP BY distinct_id
```

Comment:
I'm using offset because it has more precision than _timestamp since _timestamp doesn't consider seconds